pkg/config/config.go

```diff
@@ -41,6 +42,37 @@ type Config struct {
 	Storage          *StorageConfig
 }
 
+// Load loads the config from a file.
+// If file is not present returns default config
+func Load(configFile string) (*Config, error) {
+	_, err := os.Stat(configFile)
+	if err == nil {
+		conf, err := ParseConfigFile(configFile)
```

Comment:
if you skip this first if/else you could just return the call to `ParseConfigFile` on 56.

not a deal breaker, I just try to avoid nesting the 'happy path' in if statements.

Reply:
I _think_ `ParseConfigFile` would already return an error if the file is not found when trying to decode.